src/components/Contacts/ContactDetails/ContactDetailsTab/People/ContactPeople.graphql

```diff
@@ -14,19 +14,67 @@ fragment ContactPerson on Person {
   id
   firstName
   lastName
+  emailAddresses {
+    nodes {
+      email
+      id
+      primary
+    }
+  }
+  phoneNumbers {
+    nodes {
+      id
+      number
+      primary
+    }
+  }
+  facebookAccounts {
+    nodes {
+      id
+      value: username
```

Comment:
Let me know if you can think of a better generic name for this. I need them to be the same across all four account types, and ```value``` is what I came up with while testing.